node_package/src/clientStartup.ts

```diff
@@ -58,6 +58,13 @@ function turbolinksInstalled(): boolean {
   return (typeof Turbolinks !== 'undefined');
 }
 
+function turboInstalled() {
+  var context = findContext();
+  if (context.ReactOnRails) {
+    return context.ReactOnRails.option('turbo') === true;
+  }
+}
```

Comment:
No, I also would prefer a detection if possible. But as I noticed Turbo will be included only as module (for webpack(er)) in contrast to Turbolinks, which will inject itself to the global window.

So I don't know a way to inspect the global environment to detect if the Turbo module is used somewhere in the application.. if you have a hint for me, we can adjust that of course.

Reply:
Try require within a try block. 

I can get you a cod snippet if needed.

Reply:
The problem with detection logic like that is that it would only detect if turbo is being bundled through `webpacker` even though turbo supports `sprockets` as well.